testing/tests/DevExpress.ui.widgets.treeList/virtualScrolling.integration.tests.js

```diff
@@ -107,4 +106,104 @@ QUnit.module('Virtual Scrolling', baseModuleConfig, () => {
+        }).dxTreeList('instance');
+
+        this.clock.tick(300);
+
+        const scrollable = treeList.getScrollable();
+        const $scrollableContent = $(scrollable.content());
+        const $scrollableContainer = $(scrollable.container());
+
+        // act
+        scrollable.scrollTo({ y: 10000 });
+        $scrollableContainer.trigger('scroll');
+        this.clock.tick(1000);
+
+        // assert
+        const maxScrollTop = Math.max($scrollableContent.get(0).clientHeight - $scrollableContainer.get(0).clientHeight, 0);
+        assert.roughEqual(scrollable.scrollTop(), maxScrollTop, 1.01, 'scroll position at the end');
+    });
+
+    QUnit.test('It should be possible to scroll to the last row when there is fixed column and wordWrapEnabled is set to true', function(assert) {
+        // arrange
+        const data = generateNestedData(20, 1);
+        const done = assert.async();
+
+        data.forEach((item, index) => {
+            item.field1 = `Field1 ${index} `.repeat(2);
+        });
+        data[0].field2 = 'TestTEST TestTEST TestTEST TestTEST TestTEST TestTEST TestTEST TestTEST TestTEST TestTEST Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test Test  Test Test Test Test Test Test Test Test ';
+
+        const treeList = $('#treeList').dxTreeList({
+            dataSource: data,
+            height: 500,
+            autoExpandAll: true,
+            wordWrapEnabled: true,
+            editing: {
+                mode: 'row',
+                allowDeleting: true,
+                allowUpdating: true,
+                useIcons: true
+            },
+            columns: [{
+                dataField: 'field1',
+                width: 60,
+            }, {
+                dataField: 'field2',
+                width: 120
+            }, {
+                type: 'buttons',
+                fixed: true,
+                buttons: ['edit', 'delete']
+            }],
+            scrolling: {
+                useNative: false
+            },
+        }).dxTreeList('instance');
+
+        this.clock.tick(300);
+        this.clock.restore();
+
+        const scrollable = treeList.getScrollable();
+        const $scrollableContent = $(scrollable.content());
+        const $scrollableContainer = $(scrollable.container());
+
+        // act
+        scrollable.scrollTo({ y: 350 });
+        scrollable.scrollTo({ y: 400 });
+        scrollable.scrollTo({ y: 10000 });
+
+        setTimeout(() => {
+            // assert
+            const maxScrollTop = Math.max($scrollableContent.get(0).clientHeight - $scrollableContainer.get(0).clientHeight, 0);
+            assert.roughEqual(scrollable.scrollTop(), maxScrollTop, 1.01, 'scroll position at the end');
+            done();
+        }, 300);
```

Comment:
Let's remove real ```setTimeout``` from this test case:

```suggestion
        this.clock.tick(300);
        const maxScrollTop = Math.max($scrollableContent.get(0).clientHeight - $scrollableContainer.get(0).clientHeight, 0);
        assert.roughEqual(scrollable.scrollTop(), maxScrollTop, 1.01, 'scroll position at the end');
```

I checked this test case on my local machine with ```this.clock.tick``` and it works fine :)